AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/TestRunnerServiceImpl.kt

```diff
@@ -72,6 +74,17 @@ internal class TestRunnerServiceImpl internal constructor(
         )
     }
 
+    override suspend fun getOrUploadRemoteApk(
+        sourceGcsPath: GcsPath,
+        targetRelativePath: String,
+    ): RemoteApk {
+        val targetGcsPath = googleCloudApi.existingFilePath(targetRelativePath)?.let {
+            googleCloudApi.copy(sourceGcsPath, targetRelativePath)
+        }
+
+        return RemoteApk(gcsPath = targetGcsPath!!)
```

Comment:
why did we need `!!` ?
the expression above seems to return non-null (if not, i think it should?)

Reply:
the previous logic was wrong. updated.